src/corelibs/U2Formats/src/EMBLGenbankAbstractDocument.cpp

```diff
@@ -155,6 +155,10 @@ void EMBLGenbankAbstractDocument::load(const U2DbiRef& dbiRef, IOAdapter* io, QL
             fullSequenceSize += gapSize;
         }
 
+        if (!merge) {
+            seqImporter.resetAlphabetCharacterHit();
```

Comment:
Why don't you reset in seqImporter.startSequence()?
With your current solution you need to re-check that `resetAlphaberCharacterHit` is called in all usages of seqImporter (for all formats)

Reply:
:thumbsup: